articles/quantum-InstallConfig.md

```diff
@@ -80,7 +80,7 @@ In this section you will clone the quantum samples & libraries repository, and r
 
 #### [Command Line / Visual Studio Code](#tab/tabid-vscode)
 
-1. If you intend to use Visual Studio Code, install the [Microsoft Quantum Development Kit for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=quantum.microsoft-quantum-development-kit-vscode) extension.
+1. If you intend to use Visual Studio Code, install the [Microsoft Quantum Development Kit for Visual Studio Code](https://github.com/MicrosoftDocs/quantum-docs-pr/pull/249) extension.
```

Comment:
This link doesn't look right

Reply:
Yup, it should be https://marketplace.visualstudio.com/items?itemName=quantum.quantum-devkit-vscode.